Add information about browser compatibility to Installation guide (#8942)
withastro/docs: src/content/docs/en/install-and-setup.mdx

- **Text editor** - We recommend [VS Code](https://code.visualstudio.com/) with our [Official Astro extension](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode).
- **Terminal** - Astro is accessed through its command-line interface (CLI).

## Browser compatibility

> Astro is built with Vite, meaning it targets browsers with modern JavaScript support by default. For reference, you can see the [list of currently supported browser versions in Vite](https://vitejs.dev/guide/build.html#browser-compatibility).

Reviewer: ```suggestion
Astro is built with Vite which targets browsers with modern JavaScript support by default. For a complete reference, you can see the [list of currently supported browser versions in Vite](https://vitejs.dev/guide/build.html#browser-compatibility).
```
What do you think?

Author: Sounds good!
